packages/internal/test-util/src/lib/spy-driver/spy-driver-root.module.ts

```diff
@@ -1,38 +1,22 @@
-import { NgModule, Optional, SkipSelf } from '@angular/core';
+import { inject, NgModule, InjectFlags } from '@angular/core';
```

Comment:
nit: sort alphabetically.
```suggestion
import { inject, InjectFlags, NgModule } from '@angular/core';
```